frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -578,14 +578,15 @@ export const surveyLogic = kea<surveyLogicType>([
                     query: `
                         -- QUERYING SINGLE CHOICE RESPONSES
                         SELECT
-                            ${getResponseFieldCondition(questionIndex, question?.id)} AS survey_response,
+                            getSurveyResponse(${questionIndex}, '${question?.id}') AS survey_response,
                             COUNT(survey_response)
                         FROM events
                         WHERE event = 'survey sent'
                             AND properties.$survey_id = '${props.id}'
                             AND timestamp >= '${startDate}'
                             AND timestamp <= '${endDate}'
                             ${createAnswerFilterHogQLExpression(values.answerFilters, survey)}
+                            AND survey_response != null
```

Comment:
Does this work? Isnt 'is not null'?

Reply:
it does because of the hogQL abstraction